package.json

```diff
@@ -253,6 +257,7 @@
 		"postis": "^2.2.0",
 		"prom-client": "^12.0.0",
 		"prometheus-gc-stats": "^0.6.2",
+		"protoo-server": "^4.0.6",
```

Comment:
This package is a kind of WebSocket? Do you think it's possible to use the package we already have?

Reply:
I'm not sure which package you are referring here, `websockets` or `ws` both of them are not available here

Reply:
I think is a build-in package in meteor, but let's skip that for now